parsley/shared/src/main/scala/parsley/internal/deepembedding/singletons/TokenEmbedding.scala

```diff
@@ -41,4 +50,6 @@ private [parsley] class NonSpecific(name: String, unexpectedIllegal: String => S
     override def pretty: String = "nonspecificName"
     // $COVERAGE-ON$
     override def instr: instructions.Instr = new instructions.TokenNonSpecific(name, unexpectedIllegal)(start, letter, illegal)
+
+    override def visit[T, U[+_]](visitor: LazyParsleyIVisitor[T, U], context: T): U[String] = visitor.visit(this, context)(name, unexpectedIllegal, start, letter, illegal)
```

Comment:
break up with braces on new line

Reply:
applied also do some other impls in SymbolEmbedding due to the same issue of line length